.github/CODEOWNERS

```diff
@@ -2,5 +2,5 @@
 /stage1/ @valerydluski @sergeyshalyapin @natein
 /stage2/ @valerydluski @natein
 /node/ @AlreadyBored
-/react/ @ragingyngvarr
+/react/ @kravaring
```

Comment:
@SpaNb4 I think your GitHub is also needed here

Reply:
Oh, no